templates/postgresql.yml.erb

```diff
@@ -169,6 +169,29 @@ restapi:
     username: <%= @restapi_username %>
     password: <%= @restapi_password %>
 <% end -%>
+<% unless @ctl_authentication_username == nil && @ctl_authentication_password == nil && @ctl_insecure == nil && @ctl_cacert == nil && @ctl_certfile == nil && @ctl_keyfile == nil && @ctl_keyfile_password == nil -%>
```

Comment:
testing on exist would be enough, e.g.
unless @ctl_authentication_username && @ctl_authentication_password ...

but these checks should be done in the code directly to inform the end user vie fail